### ISSUE TITLE
any good examples for knowledge distillation using custom dataset? 

### ISSUE THREAD

**Issue by andysingal** [Deci-AI/super-gradients#1644]
### 🐛 Describe the bug

i am clueless where the datasets originated from:
```
import os
from super_gradients.training import Trainer, models, dataloaders
from super_gradients.training.metrics import Accuracy, Top5
from torchvision import transforms

checkpoint_dir = os.path.abspath("notebook_ckpts")
trainer = Trainer(experiment_name="beit_base_patch16_224_test", ckpt_root_dir=checkpoint_dir)
test_dataloader = dataloaders.get("cifar10_val", dataloader_params={"batch_size": 64, "num_workers": 2}, dataset_params={"transforms": [transforms.ToTensor(), transforms.Resize(224)]})
pretrained_beit = models.get('beit_base_patch16_224', arch_params={'num_classes': 10, "image_size": [224, 224], "patch_size": [16, 16]}, pretrained_weights="cifar10")
metrics = trainer.test(model=pretrained_beit, test_loader=test_dataloader, test_metrics_list=[Accuracy(), Top5()])
print()
print(f"Accuracy: {metrics['Accuracy']:.3f}")
print(f"Top 5:    {metrics['Top5']:.3f}")
```

### Versions

RTX 3090

**Comment by harpreetsahota204** [comment]
Hi @andysingal, SG is compatible with PyTorch dataloaders. So, you can use your own dataloaders!

Here are some in-depth docs that will help you:

 - [Data in SG](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/Data.md)
 - [Dataloader adapters](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/dataloader_adapter.md)

**Comment by andysingal** [comment]
> Hi @andysingal, SG is compatible with PyTorch dataloaders. So, you can use your own dataloaders!
> 
> Here are some in-depth docs that will help you:
> 
> * [Data in SG](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/Data.md)
> * [Dataloader adapters](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/dataloader_adapter.md)

Thank you very much
